content/blog/2024/07/kubewarden-1-15-release.md

```diff
@@ -0,0 +1,104 @@
+objects to be used in the deployment created for the PolicyServer. This
+enhancement allows users to specify tolerations directly within the
+PolicyServer CRD, simplifying the management of taints and tolerations across
+your Kubernetes cluster. By integrating this directly into the CRD, Kubewarden
+ensures a more streamlined and consistent approach to handling node
+tolerations, making it easier to deploy policies in diverse and dynamic
+environments.
+
+## Centralized Management of Tolerations, Affinity, and Anti-Affinity 
+
+In line with making the deployment process more efficient, Kubewarden v1.15
+introduces a central place where tolerations, affinity, and anti-affinity rules
+are specified and enforced across all components of the stack (controller deployment,
+audit scanner cronjob and default PolicyServer). This centralized management
+approach not only simplifies the configuration process but also ensures
+consistency and reduces the chances of configuration drift. By defining these
+rules once and applying them universally, users can achieve a more reliable and
+predictable policy enforcement setup.
+
+:::warning
+The affinity configuration previously present in the default policy server values 
+has been removed. Then, if you were using it, you need to migrate your configuration
+:::
+
+## Technical Debt Reduction in Kubewarden Controller 
+
+In v1.15, significant efforts have been made to reduce technical debt in the
+Kubewarden controller, ensuring better performance, easier maintenance, and a
+more robust foundation. Key changes include:
+
+- **Updated Golang Linter**: The stricter linter catches potential issues
+  early, ensuring higher code quality.
+- **Removed --default-policy-server CLI Flag**: The default policy server name
+  is now always "default," simplifying configuration.
+- **Updated to Kubebuilder 4.0.0**: The controller code now uses Kubebuilder
+  4.0.0.
+- **Refactored Reconciler Code**: The code is split into multiple
+  subreconcilers following the Subreconciler pattern, enhancing manageability
+  and modularity.
+- **File/Package Renaming**: Files and packages have been renamed for better
+  readability and navigation.
+
+These changes improve functionality and set the stage for more streamlined
+development and maintenance, reflecting a commitment to maintaining a
+high-quality codebase.
+
+## Policy Grouping Code Implementation 
+
+The v1.15 release marks the beginning of implementing [RFC 0020: Policy
+Group](https://github.com/kubewarden/rfc/blob/main/rfc/0020-policy-group.md),
+starting with changes to the policy server. This feature allows users to create
+complex policies by combining simpler ones, leveraging existing policies to
+avoid custom builds from scratch. The motivation is to increase reusability,
+reduce cognitive load, and enable the creation of custom policies through a
+DSL-like configuration, thus improving policy management efficiency and
+flexibility.
```

Comment:
```suggestion
flexibility.

This feature is not yet available to Kubernetes users, since we've not yet introduced
the set of new Custom Resource Definition that are required to manage policy groups.
This will be part of the next release of Kubewarden.

In the meantime, users running Kubewarden Policy Server in stand-alone mode, can
can already use this feature.

> **Note:** in case you have missed, it's possible to use Kubewarden's Policy Server
> to validate any kind of JSON payload. See [this section](https://docs.kubewarden.io/howtos/raw-policies)
> of our documentation.
```